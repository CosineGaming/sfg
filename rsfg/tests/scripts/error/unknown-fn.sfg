fn main()
	unknown(5)

