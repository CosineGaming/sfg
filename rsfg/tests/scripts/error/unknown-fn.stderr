[ERROR] called unknown function unknown at 2:2